packages/shell-chrome/views/_partials/header.edge

```diff
@@ -34,82 +35,77 @@
 
             @verbatim
             <div
-                :class="{
-                        'animate-pulse-fast': !showTools,
-                        'bg-gray-300 bg-opacity-50 text-gray-600': !showTools,
-                        'bg-green-100 text-green-700': showTools && isLatest,
-                        'bg-orange-100 text-orange-700': showTools && !isLatest,
-                    }"
                 data-testid="status-line"
-                class="flex items-center px-1 py-0.5 rounded text-xs font-medium leading-4 transition transition-colors duration-700 ease-in-out"
+                :class="{
+                    'hidden': showTools,
+                }"
+                class="flex items-center px-1 py-0.5 rounded text-xs text-gray-600 font-medium leading-4 bg-gray-300 bg-opacity-50 animate-pulse-fast transition transition-colors duration-700 ease-in-out"
                 :title="isLatest? 'Latest Version' : `Latest Version: ${latest}`"
             >
                 <svg
-                    :class="{
-                            'text-gray-400': !showTools ,
-                            'text-green-500': showTools && isLatest,
-                            'text-orange-500': showTools && !isLatest,
-                        }"
-                    class="mr-1.5 h-2 w-2 transition transition-colors duration-700 ease-in-out"
+                    class="mr-1.5 h-2 w-2 text-gray-400 transition transition-colors duration-700 ease-in-out"
                     fill="currentColor"
                     viewBox="0 0 8 8"
                 >
                     <circle cx="4" cy="4" r="3" />
                 </svg>
-                <a
-                    :class="isLatest ? 'pointer-events-none' : ''"
-                    :href="isLatest ? '#' : 'https://github.com/alpinejs/alpine/releases'"
-                    x-text="detected"
-                ></a>
+                <span x-text="loadingText"></span>
```

Comment:
Should we move this loading state to display inside the panel? I think it's fine here, but maybe there's a more obvious spot for it.

Reply:
Too much?

<img src="https://user-images.githubusercontent.com/2156070/103014047-0b981980-4536-11eb-86f7-a9edd58250cc.gif" width="200px" />

<img src="https://user-images.githubusercontent.com/2156070/103015444-5a46b300-4538-11eb-9f77-6c988b250abf.gif" width="200px" />

Reply:
I like it...

> too much?

Probably :) I'm personally okay with it though.

Reply:
I mean...it is a little too much really 😆

Just the text with a blinking cursor in the components panel might be the one.

Reply:
I'm probably not the best person to judge. Maybe let Hugo or Ryan chime in.

FWIW, I'm a big fan of Steam's loading animation:

![steamuserimages-a akamaihd](https://user-images.githubusercontent.com/1478421/103019028-b3363b80-4578-11eb-8b7a-daba7bf9d9d1.gif)

Reply:
Haha @stephenoldham let's go for blinking text

I guess we can always add a nicer animation down the line